From 8783f8d94033e82c1a9016d42e70566e4300db61 Mon Sep 17 00:00:00 2001
From: Sai Udayagiri <saibabu.udayagiri@gmail.com>
Date: Mon, 22 Jun 2026 14:01:47 +0530
Subject: [PATCH] Fixed typo in `test_atomgroup.py`

* fixed typo in test_atomgroup.py

pytest 9.1 reads a trailing comma in a parametrize argnames string as a
one-element tuple of names, so it unpacks every value: "btype," with
"bond" fails collection of TestAtomGroupToTopology ("the number of names
(1) must be equal to the number of values (4)"), and with it the whole
pytest run, on every architecture. Older pytest stripped the comma.

Upstream-Status: Backport [https://github.com/MDAnalysis/mdanalysis/commit/8783f8d94033e82c1a9016d42e70566e4300db61]
---
 testsuite/MDAnalysisTests/core/test_atomgroup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/MDAnalysisTests/core/test_atomgroup.py b/testsuite/MDAnalysisTests/core/test_atomgroup.py
index 38432e6..b285a35 100644
--- a/testsuite/MDAnalysisTests/core/test_atomgroup.py
+++ b/testsuite/MDAnalysisTests/core/test_atomgroup.py
@@ -88,7 +88,7 @@ class TestAtomGroupToTopology(object):
         assert isinstance(imp, ImproperDihedral)
 
     @pytest.mark.parametrize(
-        "btype,", ["bond", "angle", "dihedral", "improper"]
+        "btype", ["bond", "angle", "dihedral", "improper"]
     )
     def test_VE(self, btype, u):
         ag = u.atoms[:10]
